infra/bridge/src/controllers/hooks/telegram.js

```diff
@@ -80,23 +96,37 @@ router.post('/', (req, res) => {
       // Note: Username is optional in Telegram.
       // ID is returned as number, We don't want to run into the big number issues
       // So use id only if username is not set
-      const key = `telegram/follow/${member.username || member.id}`
-      redisBatch.set(key, JSON.stringify(member), 'EX', 60 * 30)
-      logger.debug(`Pushing telegram new member event to ${key}`)
+      const username = member.username || member.id
+
+      // Not awaiting this async operation intentionally
+      growthEventHelper({
+        type: 'FOLLOW',
+        socialNetwork: 'TELEGRAM',
+        username: username,
+        event: member
+      })
     })
   }
 
-  redisBatch.exec(err => {
-    if (err) {
-      logger.error(
-        `[TELEGRAM] Failed to push ${followCount}/${totalFollowEvents} new chat member events to redis`
-      )
-    } else {
-      logger.debug(
-        `[TELEGRAM] Pushed ${followCount}/${totalFollowEvents} new chat member events to redis`
-      )
-    }
-  })
+  if (totalFollowEvents > 0) {
+    logger.debug(
+      `[TELEGRAM] Processed ${followCount}/${totalFollowEvents} new chat member events`
+    )
+  }
+
+  if (shouldSendReplyMessage || shouldSendAckMessage) {
+    return res
+      .status(200)
+      .header('Content-Type', 'application/json')
+      .send({
+        method: 'sendMessage',
+        chat_id: message.chat.id,
+        text: shouldSendReplyMessage
+          ? 'Hey there, Get back to the Origin Marketplace app to continue'
+          : 'Hey there, we have received your message. Our team will reach out to you as soon as possible.'
```

Comment:
We also drop an email to the support email, in addition to storing that to DB. So everyone gets notified when an user tries to chat with the bot. We could ping them later to see what they are up to.  
  
Also, we could also build a UI so that we can reply to the messages (instead of manually pinging them). But that's not a priority right now I think.